docs_src/microservices/device/supported/device-onvif-camera/supplementary-info/api-analytic-support.md

```diff
@@ -268,7 +268,7 @@ curl --request GET 'http://0.0.0.0:59882/api/v3/device/name/Camera003/SupportedR
             "deviceName" : "Camera003",
             "id" : "6fca707b-3c52-4694-be37-2e23ecf65de1",
             "objectValue" : {
-               "SupportedRules" : {
+               "AnalyticsSupportedRules" : {
```

Comment:
this is a tricky one. i _believe_ it should not be changed, as it is the schema for the response from the camera and not from edgex

Reply:
yep. everything that is in `objectValue` comes direct from the camera and should not be changed.

![image](https://github.com/edgexfoundry/edgex-docs/assets/48735240/58588e32-701a-4e4a-95c4-904a38b07305)